integrations-npm/ts4-react17/package.json

```diff
@@ -6,22 +6,31 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "test": "react-scripts test"
   },
   "dependencies": {
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^12.1.5",
+    "@types/d3-scale": "4.0.9",
+    "d3-scale": "4.0.2",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-scripts": "^5.0.0",
-    "recharts": "^3.0.0-alpha.8",
-    "d3-scale": "4.0.2",
-    "@types/d3-scale": "4.0.9"
+    "recharts": "^3.0.0-alpha.8"
```

Comment:
Hold on, this is alpha.8, this is before the es-toolkit PR, isn't it?

Reply:
Hm from the logs: Dependency recharts is installed and only one version is present: 3.0.0-beta.1 so it installs beta.1 and seems to ignore this package.json? Why?